src/commands/config.rs

```diff
@@ -1,27 +1,51 @@
 use crate::terminal::message;
+#[cfg(test)]
+use std::env;
 use std::fs;
-use std::path::Path;
+#[cfg(not(target_os = "windows"))]
+use std::os::unix::fs::PermissionsExt;
+use std::path::{Path, PathBuf};
 
 use crate::emoji;
+use crate::install::target;
 use crate::settings::global_user::GlobalUser;
 
-pub fn global_config(email: &str, api_key: &str) -> Result<(), failure::Error> {
+static CONFIG_PERM: u32 = 0o400;
```

Comment:
People may want to edit their config. Probably better to make this `600`, right?

Reply:
That's a good point, I believe I already did it too.